README.md

```diff
@@ -14,7 +14,17 @@ The `cardano-cli` tool allows users to perform various operations and actions on
 
 ## Executables
 
-You can download the hydra binaries of ``cardano-node`` and ``cardano-cli`` from the [release notes](https://github.com/IntersectMBO/cardano-node/releases)
+You can download the binaries of ``cardano-node`` and ``cardano-cli`` from the [release notes](https://github.com/IntersectMBO/cardano-node/releases)
```

Comment:
```suggestion
You can download the binaries of `cardano-cli` compatible with specific versions of `cardano-node` from [cardano-node's release notes](https://github.com/IntersectMBO/cardano-node/releases).

You can also download binaries of all versions of `cardano-cli` from [cardano-cli's release notes](https://github.com/IntersectMBO/cardano-cli/releases)
```

Optional

Reply:
Good idea, thanks! I'll add a compatibility warning for cli releases as well.